Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FINUFFT"
 uuid = "d8beea63-0952-562e-9c6a-8e8ef7364055"
 author = "Ludvig af Klinteberg <ludvigak@kth.se>"
```

Comment:
do you still get email at KTH? :)

Reply:
Haha, yes so far they haven't shut down the address :D